api/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: "2.0"
+info:
+  description: "Just a dictionary for **\"Галицька\"** language."
+  version: "1.0.0"
+  title: "Dictionary"
+externalDocs:
+  description: "GitHub"
+  url: "https://github.com/yushni/dictionary"
+host: "localhost"
+basePath: "/"
+tags:
+- name: "word"
+  description: ""
```

Comment:
description by default - "", we don't need to override the default value by new default value :)